synapseutils/sync.py

```diff
@@ -334,11 +342,13 @@ def syncToSynapse(syn, manifestFile, dryRun=False, sendMessages=True, retries=MA
 
     """
     df = readManifestFile(syn, manifestFile)
-    sizes = [os.stat(os.path.expandvars(os.path.expanduser(f))).st_size for f in df.path if not is_url(f)]
-    #Write output on what is getting pushed and estimated times - send out message.
-    sys.stdout.write('='*50+'\n')
-    sys.stdout.write('We are about to upload %i files with a total size of %s.\n ' %(len(df), utils.humanizeBytes(sum(sizes))))
-    sys.stdout.write('='*50+'\n')
+
+    if 'dataFileHandleId' not in df.columns:
```

Comment:
what about a case with a mix of dataFileHandleId and path? will nothing be printed but we still perform uploads?